shelter/remoteattestation/securitychannel.go

```diff
@@ -75,22 +80,30 @@ func EnclaveTlsSetupTcpSock(ipaddress string, port string, logLevelInit string,
 	} else if strings.EqualFold(logLevelInit, "off") {
 		logLevel = C.ENCLAVE_TLS_LOG_LEVEL_NONE
 	}
-	ret := C.ra_tls_echo(C.int(sockfd.Fd()), C.enclave_tls_log_level_t(logLevel), C.CString(attester), C.CString(verifier), C.CString(tls), C.CString(crypto), C.bool(mutual))
+	ret := C.ra_tls_echo(C.int(sockfd.Fd()), C.enclave_tls_log_level_t(logLevel), C.CString(attester), C.CString(verifier), C.CString(tls), C.CString(crypto), C.bool(mutual), (*C.uchar)((unsafe.Pointer)(&sendMsg[0])), (C.uint)(sendMsgLen), (*C.uchar)((unsafe.Pointer)(&utils.ReceiveMsg[0])), (*C.uint)((unsafe.Pointer)(&utils.ReceiveMsgLen)))
```

Comment:
Please split this single line into multiple lines.

Reply:
done